_calls/fields-search.md

```diff
@@ -0,0 +1,532 @@
+---
+layout: call
+priority: 4
+title:  Recherche personnalisée
+description: > #
+ Le variable <b>fields</b> vous permet de filter les informations.
+
+ <h3 class="ui header">Champs possibles:</h3>
+ <ul class="ui list">
+   <li>code</li>
+   <li>nom</li>
+   <li>codesPostaux</li>
+   <li>surface</li>
+   <li>population</li>
+   <li>centre</li>
+ </ul>
```

Comment:
If you have that much information, I doubt it should be in the YAML front-matter. Where are you interpolating `page.description`?